src/platform/common/application/extensions.web.ts

```diff
@@ -28,6 +28,7 @@ export class Extensions implements IExtensions {
     public async determineExtensionFromCallStack(): Promise<{ extensionId: string; displayName: string }> {
         const stack = new Error().stack;
         if (stack) {
+            const jupyterExtRoot = this.getExtension(JVSC_EXTENSION_ID)!.extensionUri.toString();
```

Comment:
Doesn't this need to be lower cased?

Reply:
why is that?

Reply:
Because you compare it to lower case values below.

Reply:
Fixed